frontend/src/components/Login/Login.css

```diff
@@ -0,0 +1,93 @@
+.login {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.login-card {
+  padding: 3em;
+  display: flex;
+  margin-top: 0.2rem;
+  align-self: center;
+  max-width: 350px;
+  padding: 1.5em;
+  border: 1px solid var(--white-shade-2);
+  width: 100%;
+  flex-direction: column;
```

Comment:
you have some repetitive css like `display-flex` instead you can move common css to a generic css file with a common class to avoid repeating styles again and again